swim/node.go

```diff
@@ -510,3 +510,7 @@ func (n *Node) pingNextMember() {
 		"numErrors": len(errs),
 	}).Warn("ping request inconclusive due to errors")
 }
+
```

Comment:
agreed re: the local member. That was the only nuance that felt out of place when using Pingable. I like Reachable a lot. Let's go with it.